src/scripts/spot.js

```diff
@@ -358,20 +403,20 @@ module.exports = function (robot) {
     Support = require('./support/spotifySupport')(robot, URL, Queue);
     Assoc = require('./support/spotifyAssoc')(robot);
 
-    //if (process.env.HUBOT_SPOTIFY_PLAYLIST_FILE) {
-    //    // Set up default queue
-    //    setupDefaultQueue(playlistQueue);
-    //
-    //    // Set the default queue on the queue master
-    //    queueMaster.setDefault(playlistQueue);
-    //
-    //    // Add the user queue
-    //    queueMaster.addQueue(Queue);
-    //
-    //    // Conduct the queues (the default queue will
-    //    // play if user queue is empty)
-    //   queueMaster.conduct();
-    //}
+    if (process.env.HUBOT_SPOTIFY_PLAYLIST_ID) {
+        // Set up default queue
+        setupDefaultQueue(playlistQueue);
+
+        // Set the default queue on the queue master
+        queueMaster.setDefault(playlistQueue);
+
+        // Add the user queue
+        queueMaster.addQueue(Queue);
+
+        // Conduct the queues (the default queue will
+        // play if user queue is empty)
+       queueMaster.conduct();
```

Comment:
I am not certain, but I believe part of the reason this was commented out was `conduct` instantly plays the next track - (interrupting anything that is currently playing on restart of dibsy)